code/game/objects/items/storage/secure.dm

```diff
@@ -28,31 +28,29 @@
 
 /obj/item/storage/secure/examine(mob/user)
 	..()
-	to_chat(user, text("The service panel is [src.open ? "open" : "closed"]."))
+	to_chat(user, text("The service panel is currently <b>[open ? "unscrewed" : "screwed shut"]</b>."))
 
 /obj/item/storage/secure/attackby(obj/item/W, mob/user, params)
 	if(locked)
 		if (istype(W, /obj/item/screwdriver))
-			if (do_after(user, 20*W.toolspeed, target = src))
-				src.open =! src.open
-				user.show_message("<span class='notice'>You [open ? "open" : "close"] the service panel.</span>", 1)
+			if (do_after(user, 20*W.toolspeed, target = user))
+				open =! open
+				to_chat(user, "<span class='notice'>You [open ? "open" : "close"] the service panel.</span>")
 			return
-		if ((istype(W, /obj/item/device/multitool)) && (src.open == 1)&& (!src.l_hacking))
-			user.show_message("<span class='danger'>Now attempting to reset internal memory, please hold.</span>", 1)
-			src.l_hacking = 1
-			if (do_after(usr, 100*W.toolspeed, target = src))
-				if (prob(33))
-					src.l_setshort = 1
+		if (istype(W, /obj/item/wirecutters) || istype(W, /obj/item/card/emag))
+			to_chat(user, "<span class='danger'>[src] is protected from this sort of tampering, perhaps a <b>multitool</b> could pulse the wires controlling internal memory instead?</span>")
+		if ((istype(W, /obj/item/device/multitool)) && (!l_hacking))
+			if(src.open == 1)
+				to_chat(user, "<span class='danger'>Now attempting to reset internal memory, please hold.</span>")
+				src.l_hacking = 1
+				if (do_after(usr, 400*W.toolspeed, target = user))
+					to_chat(user, "<span class='danger'>Internal memory reset - lock has been disengaged.</span>")
 					src.l_set = 0
-					user.show_message("<span class='danger'>Internal memory reset.  Please give it a few seconds to reinitialize.</span>", 1)
-					sleep(80)
-					src.l_setshort = 0
 					src.l_hacking = 0
 				else
-					user.show_message("<span class='danger'>Unable to reset internal memory.</span>", 1)
 					src.l_hacking = 0
 			else
-				src.l_hacking = 0
+				to_chat(user, "<span class='notice'>You must <b>unscrew</b> the service panel before you can pulse the wiring.</span>")

```

Comment:
I'm also not mad about just giving away the whole process in a single message. Discovery *is* part of the fun, even if we're making things less arcane with examine hints.

Reply:
It isnt a hint when you litteraly put it in italatics or bold what you need to do

Reply:
There was a player complaint that the existing method wasn't obvious and necessitated wiki diving.

As we're already in the process of adding these hints, it seemed an obvious use case where something is edge enough that most people don't know it.

Reply:
>Encouraging goon

Reply:
Just replace this by "You can't pulse the wires while the service panel is screwed."